bundle/edu.gemini.itc/src/main/java/edu/gemini/itc/flamingos2/Flamingos2Recipe.java

```diff
@@ -327,8 +329,8 @@ public ImagingResult calculateImaging() {
         return ImagingResult.apply(p, instrument, IQcalc, SFcalc, peak_pixel_count, IS2Ncalc);
     }
 
-    public double getExposureTime() {
-        return exposureTime;
+    public int getExposureTime() {
+        return (int) exposureTime;
```

Comment:
> updates the F2 ITC to return the exposure time rounded up to an integer

Maybe it is unimportant but wouldn't this round down?  7.9 seconds would become 7 seconds.